internal/pkg/daemon/bpfrecorder/bpfrecorder_apparmor.go

```diff
@@ -203,6 +203,27 @@ func (b *AppArmorRecorder) handleCapabilityEvent(capEvent *bpfEvent) {
 	b.recordedCapabilities[mid] = append(b.recordedCapabilities[mid], requestedCap)
 }
 
+// Delete all data recorded for a particular mount namespace.
+//
+// The recorder triggers this after container initialization to make sure that
+// permissions needed for setup are not included in the final profile.
+func (b *AppArmorRecorder) clearMntns(event *bpfEvent) {
+	// Everything we captured so far is container initialization.
```

Comment:
nit: This comment is not so relevant if this function gets called from a different context.